### ISSUE TITLE
JPEG CAR(Compression Artifacts Removal) for Colored Images

### ISSUE THREAD

**Issue by kewlcoder** [JingyunLiang/SwinIR#79]
Does Swin_IR also work for colored JPEG images for JPEG_CAR problem? If yes, how to use it?

**Comment by JingyunLiang** [comment]
No. We haven't trained yet. We just followed the common setting in JPEG_CAR. React with thumbs up here if you want such a model. I will train and release one if there are more than 15 thumbs up. Or you can train one yourself with a few code modification.

**Comment by rituational** [comment]
Hi @JingyunLiang 

It will be great if SwinIR can support CAR on color images. From the likes, it looks like many people are interested in the same. Let me know if you have plans to retrain the model to support this. It will be a great help.
Thanks

**Comment by JingyunLiang** [comment]
Great! I will train it today. Do you want four separate models like 
[006_CAR_DFWB_s126w7_SwinIR-M_jpeg10.pth](https://github.com/JingyunLiang/SwinIR/releases/download/v0.0/006_CAR_DFWB_s126w7_SwinIR-M_jpeg10.pth)
[006_CAR_DFWB_s126w7_SwinIR-M_jpeg20.pth](https://github.com/JingyunLiang/SwinIR/releases/download/v0.0/006_CAR_DFWB_s126w7_SwinIR-M_jpeg20.pth)
[006_CAR_DFWB_s126w7_SwinIR-M_jpeg30.pth](https://github.com/JingyunLiang/SwinIR/releases/download/v0.0/006_CAR_DFWB_s126w7_SwinIR-M_jpeg30.pth)
[006_CAR_DFWB_s126w7_SwinIR-M_jpeg40.pth](https://github.com/JingyunLiang/SwinIR/releases/download/v0.0/006_CAR_DFWB_s126w7_SwinIR-M_jpeg40.pth)

Or a joint model for JPEG10~40?

**Comment by rituational** [comment]
I think separate will be better if the quality of restored images is better that way. Do you think having separate models will help?

**Comment by JingyunLiang** [comment]
Yes, I agree. Training separate models have best performance. I will update here when the RGB models are released.

**Comment by Siannodel** [comment]
I'm also looking forward to it! Thanks!

**Comment by rituational** [comment]
Hi @JingyunLiang, any updates on this? Thanks

**Comment by JingyunLiang** [comment]
See results for Colored JPEG Compression Artifacts Removal in https://github.com/JingyunLiang/SwinIR#results and pretrained models at https://github.com/JingyunLiang/SwinIR/releases.

**Comment by rituational** [comment]
Hi, 
Thanks again @JingyunLiang  for adding this. A minor thing. The testsets/LIVE folder doesn't exist. It is being called in the sample code you provided: 
python main_test_swinir.py --task color_jpeg_car --jpeg 10 --model_path model_zoo/swinir/006_colorCAR_DFWB_s126w7_SwinIR-M_jpeg10.pth --folder_gt testsets/LIVE1
but it gives error that no image found since the folder is empty.

**Comment by rituational** [comment]
Hi @JingyunLiang, you have provided small, medium and large variants for super resolution. For CAR, was there any specific reason for using the medium variant? (If I am not wrong, you have not provided small version right?) Thanks.
